README.md

```diff
@@ -5,75 +5,46 @@
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-AsyncAPI Model SDK is a set of classes/functions for generating data models from JSON Schema and AsyncAPI spec.
+Modelina is used to generate different data models, such as Java, TypeScript classes, Go Structs, etc, for AsyncAPI message payloads among other supported inputs.
 
 ---
 
 ## :loudspeaker: ATTENTION:
 
-This package is under development and it has not reached version 1.0.0 yet, which means its API might get breaking changes without prior notice. Once it reaches its first stable version, we'll follow semantic versioning.
+This package is under development and it has not reached version 1.0.0 yet, which means its API might contain breaking changes without prior notice. Once it reaches its first stable version, we'll follow semantic versioning. So please use tests to ensure expected output or hardcode version used.
 
 ---
 
 <!-- toc is generated with GitHub Actions do not remove toc markers -->
 
 <!-- toc -->
 
-- [Installation](#installation)
-- [How it works](#how-it-works)
-  * [The input process](#the-input-process)
-  * [The generation process](#the-generation-process)
-- [Example](#example)
-- [Supported input](#supported-input)
-  * [AsyncAPI input](#asyncapi-input)
-  * [JSON Schema input](#json-schema-input)
+- [Usage](#usage)
+- [Features](#features)
+- [Roadmap](#roadmap)
 - [Customization](#customization)
 - [Development](#development)
 - [Contributing](#contributing)
 - [Contributors ✨](#contributors-%E2%9C%A8)
 
 <!-- tocstop -->
 
-## Installation
-
+## Usage
 Run this command to install the SDK in your project:
 
 ```bash
 npm install --save @asyncapi/modelina
 ```
 
-## How it works
-
-The process of creating data models from input data consists of 2 processes, the input and generation process.
-
-### The input process
-
-The input process ensures that any supported input is handled correctly, the basics are that any input needs to be converted into our internal model representation `CommonModel`. The following inputs are supported:
-
-- [JSON Schema Draft 7](#JSON-Schema-input), this is the default inferred input if we cannot find another input processor.
-- [AsyncAPI version 2.0.0 and 2.1.0](#AsyncAPI-input)
-
-Read more about the input process [here](./docs/input_processing.md).
-
-### The generation process
-
-The generation process uses the predefined `CommonModel`s from the input stage to easily generate models regardless of input. 
-The list of supported output languages can be found [here](./docs/generators.md#supported-languages).
-
-Check out [the example](#example) to see how to use the library and [generators document](./docs/generators.md) for more info.
-
-> **NOTE**: Each implemented language has different options, dictated by the nature of the language.
-
-## Example
-
+You can then choose between all the generators, for example using the TypeScript generator, as in this example:
```

Comment:
Left out `A generator is a unique kind of function that returns iterator objects with a sequence of values instead of returning single values.` as I feel like it needs more reasoning behind it. This will be explained in `./docs/usage.md#understanding-the-output-format`